tests/extensions/test_version.py

```diff
@@ -140,6 +140,59 @@ def test_full_copy(self):
         self.assertIs(successor, item1_copy)
         self.assertIs(latest, item1_copy)
 
+    def test_setting_none_clears_link(self):
+        deprecated = False
+        latest1 = make_item(2013)
```

Comment:
Why `latest1`?  Why not use just `latest` here and below for latest, predecessor, and successor in ` test_setting_none_clears_link` for Item and Collection tests?  It makes sense in `test_multiple_link_setting` but had me looking for something I had missed in this test.

Reply:
Copy pasta mistake. Good catch!